lib/src/multipart_request.dart

```diff
@@ -124,8 +126,9 @@ class MultipartRequest extends BaseRequest {
   ///
   /// [RFC 2046]: http://tools.ietf.org/html/rfc2046#section-5.1.1.
   /// [RFC 1521]: https://tools.ietf.org/html/rfc1521#section-4
-  static const List<int> _BOUNDARY_CHARACTERS = const <int>[
-    39, 43, 95, 44, 45, 46, 58, 61, 63, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57,
+  @visibleForTesting
+  static const List<int> BOUNDARY_CHARACTERS = const <int>[
```

Comment:
Better idea

1) Put this in another file in src
2) import here
3) import via `lib/src/...` in the test
4) Don't expose another top-level field

I was going to say exclude from 'lib/http.dart' but that's more error prone